storage/gcp/gcp.go

```diff
@@ -73,9 +72,17 @@ type objStore interface {
 
 // coord describes a type which knows how to sequence entries.
 type sequencer interface {
+	// assignEntries should durably allocate contiguous index numbers to the provided entries,
+	// and return the lowest assigned index.
 	assignEntries(ctx context.Context, entries [][]byte) (uint64, error)
+	// consumeEntries should call the provided function with up to limit previously sequenced entries.
+	// If the call to consumeFunc returns no error, the entries should be considered to have been consumed.
+	consumeEntries(ctx context.Context, limit uint64, f consumeFunc) (bool, error)
```

Comment:
The doc comment doesn't tell the purpose of the returned `bool` although it was mentioned in the actual implementation doc comment.

Reply:
Good spot, added - thanks!